Makefile

```diff
@@ -80,8 +80,9 @@ tgi_server:
 	VERSION=${VERSION} TGI_VERSION=${TGI_VERSION} make -C text-generation-inference/server gen-server
 
 tgi_test: test_installs tgi_server
-	find text-generation-inference -name "text_generation_server-$(VERSION)-py3-none-any.whl" \
-	                               -exec python -m pip install --force-reinstall {} \;
+#	find text-generation-inference -name "text_generation_server-$(VERSION)-py3-none-any.whl" \
+#	                               -exec python -m pip install --force-reinstall {} \;
+	python -m pip install --force-reinstall /tgi/build/dist/text_generation_server-$(VERSION)-py3-none-any.whl
```

Comment:
this will break installation outside docker. 
Why did you do this change? The target depends on tgi_server anyway, that should have built the given version on the expected path.